core_data_modules/data_models/scheme.py

```diff
@@ -2,19 +2,23 @@
 
 
 class Scheme(object):
-    scheme_id = None
-    name = None
-    version = None
-    codes = []
-    documentation = dict()
+    def __init__(self, scheme_id, name, version, codes, documentation=None):
+        if documentation is None:
+            documentation = dict()
+
+        self.scheme_id = scheme_id
+        self.name = name
+        self.version = version
+        self.codes = codes
+        self.documentation = documentation
 
     def __eq__(self, other):
         if not isinstance(other, self.__class__):
             return False
-        return other.scheme_id == self.scheme_id & \
-            other.name == self.name & \
-            other.version == self.version & \
-            other.documentation == self.documentation & \
+        return other.scheme_id == self.scheme_id and \
```

Comment:
Is this a bug fix, or a style change?

Reply:
Bug fix. As explained yesterday, & takes precedence over equals. If you wanted to keep the (in my opinion not Pythonic) &s you'd need to add brackets

Reply:
ack